apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/FaucetButton.tsx

```diff
@@ -1,5 +1,6 @@
 "use client";
 
+import { ChakraProviderSetup } from "@/components/ChakraProviderSetup";
```

Comment:
Chakra? I thought we were done with Chakra?

Reply:
Not even close lol. Might take another few weeks